documentation/modules/proc-accessing-kafka-using-ingress.adoc

```diff
@@ -45,22 +52,33 @@ spec:
   zookeeper:
     # ...
 ----
+<1> Ingress hosts for the bootstrap service and kafka brokers.
 
-. Make sure the hosts in the `configuration` section properly resolve to the Ingress endpoints.
+. Make sure the hosts in the `configuration` resolve to the Ingress endpoints.
++
+[source,shell,subs=+quotes]
+kubectl get services
+kubectl describe service _INGRESS-NAME_
```

Comment:
@scholzj - Not sure what this step is meant to do. So added some placeholder commands!

Reply:
This whole segment should be removed - I do not think there is any easy way to check the routing of the DNS names to the Ingress.

Reply:
Removed the step